src/components/themed/SimpleTextInput.tsx

```diff
@@ -187,6 +197,11 @@ export const SimpleTextInput = React.forwardRef<SimpleTextInputRef, SimpleTextIn
     return disabled ? theme.textInputPlaceholderColorDisabled : isFocused ? theme.textInputPlaceholderColorFocused : theme.textInputPlaceholderColor
   }, [disabled, isFocused, theme.textInputPlaceholderColor, theme.textInputPlaceholderColorDisabled, theme.textInputPlaceholderColorFocused])
 
+  React.useEffect(() => {
+    if (selected == null) return
+    focusAnimation.value = selected ? withTiming(1, { duration: baseDuration }) : withDelay(animationDelay, withTiming(0, { duration: baseDuration }))
+  }, [selected, focusAnimation, baseDuration, animationDelay])
```

Comment:
Would `useDerivedValue` be the right choice of a hook to bind `focusAnimation` to the `selected` (hopefully `active`) prop?

Reply:
Keeping in mind the `null` case noop is necessary, then it would error about using its own value in its initializer if I converted this into `useDerivedValue`